server/src/Core/Interfaces/IGameNameDuplicationsChecker.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+namespace BGS.ApplicationCore.Interfaces;
+
+public interface IGameNameDuplicationsChecker
```

Comment:
rename to `IGameDuplicationChecker`.
In this case I don't think we should focus on implementation details in the interface name